app/assets/javascripts/tours.js

```diff
@@ -0,0 +1,25 @@
+/* global Tour */
+
++function () {
+  $(document).on('click', '[data-tour]', function (event) {
+    event.preventDefault()
+    event.stopPropagation()
+
+    var $link        = $(this)
+    var tourDefaults = {
+      steps:    $link.data('tour'),
+      template: $('[data-tour-template]').html()
+    }
+
+    if ($link.data('steps').length == 1) {
```

Comment:
Ahh, se me pasó esto:
```suggestion
    if ($link.data('steps').length === 1) {
```

Reply:
habría que agregarlo al eslint ;-)

Reply:
Si, de una, ahí lo pongo.